### PR TITLE
update future apis

### PR DESCRIPTION
**Pull request by halink0803**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Binance Client For Binance API
+
+source api docs: https://binance-docs.github.io
```
Comment (NNagato) [inline README.md]: pls replace this link to https://binance-docs.github.io/apidocs, current link lead to 404 page

File: future.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+)
+
+const (
+	futureAPIBaseURL = "https://fapi.binance.com"
+)
+
+// CreateFutureOrder ...
+func (bc *Client) CreateFutureOrder(symbol, side, positionSide, tradeType string, quantity float64) (CreateOrderResult, error) {
+	var (
+		response CreateOrderResult
```
Comment (NNagato) [inline future.go]: this response type is for spot, future API return more data https://binance-docs.github.io/apidocs/futures/en/#new-order-trade. the param tradeType is not used in this function and refer from the docs there are more params can be supported in this api

File: future.go [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+	AvailableBalance   string `json:"availableBalance"`
+	MaxWithdrawAmount  string `json:"maxWithdrawAmount"`
+	MarginAvailable    bool   `json:"marginAvailable"`
+	UpdateTime         uint64 `json:"updateTime"`
+}
+
+// FutureAccountBalance ...
+func (bc *Client) FutureAccountBalance() (FutureAccountBalance, error) {
+	var (
+		response FutureAccountBalance
```
Comment (NNagato) [inline future.go]: this api return an array

**Review by NNagato — CHANGES_REQUESTED** [review]

File: future.go [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+	"strconv"
+)
+
+const (
+	futureAPIBaseURL = "https://fapi.binance.com"
+)
+
+// CreateFutureOrder ...
+func (bc *Client) CreateFutureOrder(symbol, side, positionSide, tradeType, timeInForce, reduceOnly, newClientOrderID, closePosition, workingType, priceProtect, newOrderRespType string,
+	price, stopPrice, activationPrice, callbackRate, quantity float64) (CreateOrderResult, error) {
```
Comment (NNagato) [inline future.go]: CreateOrderResult is for spot order, please create a new type for future order

File: future.go [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+	MarkPrice        string `json:"markPrice"`
+	MaxNotionalValue string `json:"maxNotionalValue"`
+	PositionAmt      string `json:"positionAmt"`
+	Symbol           string `json:"symbol"`
+	UnrealizedProfit string `json:"unRealizedProfit"`
+	PositionSide     string `json:"positionSide"`
+}
+
+// GetPositionInformation ...
+func (bc *Client) GetPositionInformation() ([]PositionInformation, error) {
```
Comment (NNagato) [inline future.go]: this API has symbol param as optional

**Review by NNagato — CHANGES_REQUESTED** [review]

**Comment by halink0803** [comment]
@NNagato pls re-check it.

**Review by NNagato — APPROVED** [review]
LGTM

**Pull request merged**